apps/backend/src/modules/ai-assistant/libs/helpers/change-task/geregate-change-task-prompt.ts

```diff
@@ -0,0 +1,22 @@
+import {
```

Comment:
Fix a type in file name

Reply:
I hadn't found a plugin for WebShtorm to check file/path spelling.

Reply:
So, I struggle with this.